workbench-core/authentication/src/userManagementService.ts

```diff
@@ -3,6 +3,16 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
+// disabling because the tsdoc links need the imports to work
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { IdpUnavailableError } from './errors/idpUnavailableError';
```

Comment:
I noticed that `createUser` throws four possible exceptions, but you are importing seven new error classes. Are the extra three error classes for previously documented methods, are imports that are actually not used, or is there a missing update to other methods' `@throws` lists?